packages/babel-parser/test/helpers/runFixtureTests.js

```diff
@@ -221,22 +189,43 @@ function runTest(test, parseFunction) {
   }
 }
 
-function serialize(value) {
+function serialize(key, value) {
   if (typeof value === "bigint") {
```

Comment:
Does it imply that the serialization result is dependent on node versions? Because `BigInt` is not supported in Node.js 6 and 8.

I am surprised that they do not bail.

Reply:
When the AST should contain a bigint, `null` is used instead. For this reason, I made the serialisation dependent on BigInt support.

This is also true before this PR: if you delete `packages/babel-parser/test/fixtures/estree/bigInt/basic/output.json` and run the tests on node 6, you'll get a different output.